bodhi/server/services/updates.py

```diff
@@ -19,6 +19,8 @@
 from sqlalchemy import func, distinct
 from sqlalchemy.sql import or_
 
+from pyramid.exceptions import HTTPForbidden
```

Comment:
For [PEP-8](https://www.python.org/dev/peps/pep-0008/#imports), this import should be grouped with the other third party imports (like ```sqlalchemy```).